### PR TITLE
DON-361 upgrade to angular 11

### PR DESCRIPTION
**Pull request by tbg-jasonfung**
- DON-361 upgrade to angular 11

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,18 +24,18 @@
     "prerender": "ng run donate-frontend:prerender"
   },
   "dependencies": {
-    "@angular/animations": "^10.2.3",
+    "@angular/animations": "^11.0.5",
     "@angular/cdk": "^10.2.7",
```
Comment (NoelLH) [inline package.json:28]: v11?

File: package.json [hunk 2]
```diff
@@ -24,18 +24,18 @@
...
-    "@angular/common": "^10.2.3",
-    "@angular/compiler": "^10.2.3",
-    "@angular/core": "^10.2.3",
+    "@angular/common": "^11.0.5",
+    "@angular/compiler": "^11.0.5",
+    "@angular/core": "^11.0.5",
     "@angular/flex-layout": "^10.0.0-beta.32",
-    "@angular/forms": "^10.2.3",
+    "@angular/forms": "^11.0.5",
     "@angular/material": "^10.2.7",
```
Comment (NoelLH) [inline package.json:34]: v11? This should have its own `ng update` and you should get instructions on the Update guide subdomain if you select that the project uses Material.

File: package.json [hunk 3]
```diff
@@ -56,10 +56,10 @@
...
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.1002.0",
-    "@angular/cli": "^10.2.0",
-    "@angular/compiler-cli": "^10.2.3",
-    "@angular/language-service": "^10.2.3",
+    "@angular-devkit/build-angular": "^0.1100.5",
+    "@angular/cli": "^11.0.5",
+    "@angular/compiler-cli": "^11.0.5",
+    "@angular/language-service": "^11.0.5",
     "@nguniversal/builders": "^10.1.0",
```
Comment (NoelLH) [inline package.json:63]: v11

**Review by NoelLH — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (tbg-jasonfung) [reply package.json]: Ahhh I see, I've gone ahead and done this now

**Review by NoelLH — APPROVED** [review]

**Pull request merged**